### ISSUE TITLE
training time and backward() question

### ISSUE THREAD

**Issue by JohnnieXDU** [lliuz/UnRigidFlow#2]
Hello, 

thanks for sharing the code! Appreciate it!
I have two questions refer to your implementation.

1) Your training includes three steps: flow, depth and joint estimation. So, how long does each training takes from scratch?

2) In your code, i don't understand why the loss should be multiplied by 1024, then the gradients are divided by 1024. These confusing codes are in each trainer file, for example, 47-53 lines in kitti_flow_trainer.py:

```
scaled_loss = 1024. * loss
scaled_loss.backward()

for param in [p for p in self.model.parameters() if p.requires_grad]:
                param.grad.data.mul_(1. / 1024)

self.optimizer.step()
```

3) Though the models are warped with DataParallel, but it seems doesn't work for multi-gpu. Is it true? or it's my computer's issue. I have 4 GPUs.

Thanks!
